server.go

```diff
@@ -1,174 +1,233 @@
 package main
 
 import (
-    "log"
-    "net/http"
-    "sync"
-    "os"
-    "fmt"
-    "gopkg.in/yaml.v2"
-
-    "github.com/garyburd/redigo/redis"
-    "github.com/gorilla/websocket"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"sync"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/garyburd/redigo/redis"
+	"github.com/gorilla/websocket"
 )
 
 type Config struct {
-    Redis struct {
-        Port string `yaml:"port"`
-        Pass string `yaml:"pass"`
-    } `yaml:"redis"`
-    Server struct {
-        Port string `yaml:"port"`
-    } `yaml:"server"`
+	Redis struct {
+		Address string `yaml:"port"`
```

Comment:
Perhaps this yaml config should have other name now, such as `address`

Reply:
Yes, I will do it!